Automattic/wp-calypso
# Add/site icon component

#### Proposed Changes

Integrate site upload icon component, so now Link In Bio and Newsletter are using the same component for icon upload.

#### Testing Instructions

- Run `yarn start` from the calypso root folder
- Navigate to http://calypso.localhost:3000/setup/linkInBioSetup?flow=link-in-bio&siteSlug=YOUR-SITE.wordpress.com
- You should be able to upload an Icon to your link in bio setup site.


### Expected result
- You should see the image upload, similar to this:

<img width="636" alt="image" src="https://user-images.githubusercontent.com/2653810/183027828-a065dbbb-074e-4710-8a2f-4a0b3847fb8f.png">


* Form validation and other fields are not part of this PR.


Related to #66289
Fixes #66289
